CODING.md

```diff
@@ -0,0 +1,59 @@
+# Coding guide
+
+This project is written in Go programming language and follows a community accepted coding styles and practices defined in https://golang.org/doc/effective_go.html and https://github.com/golang/go/wiki/CodeReviewComments. This document defines additional guidelines specific to the Bee project.
+
+Developers should keep their Go tooling up to date with the latest stable versions.
+
+## Static code analysis
+
+Linting is done by using https://github.com/golangci/golangci-lint with configuration defined in the root of the project and also `go vet`. Executing `make lint vet` should pass without any warnings or errors.
```

Comment:
i am not up to speed why the CI on `bee`, but i run the linter in `swarm` through:

`golangci-lint run --tests --deadline=2m --disable-all --enable=goimports --enable=varcheck --enable=vet --enable=gofmt --enable=misspell --enable=goconst ./...`

is this essentially the same? if not: why was this changed?

Reply:
The makefile is there to make the command easy to run and to be able to update it if needed without communicating the change. In bee, we are using the default configuration, not a subset like in `swarm`. But with makefile, we can change it or add new commands.